spec/version.dd

```diff
@@ -537,6 +538,127 @@ INT!(17) c;  // error, static assert trips
 	)
 	)
 
+$(H2 $(LNAME2 staticforeach, Static Foreach))
```

Comment:
there's an error here i think because of the case

Reply:
`StaticForeach`

Reply:
There is also this line in the file, from which it is derived.
`$(H2 $(LNAME2 staticif, Static If Condition))`

I don't think whatever error there is is actually in this line.

Reply:
Yeah, i may have misinterpreted the log, sorry.